# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/util/APZThreadUtils.cpp
# Commit: 16c5dd2cfd60
# Full Hash: 16c5dd2cfd600a62c80ca17cfb35c0868e016056
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r ca8139d58d58 -r 16c5dd2cfd60 gfx/layers/apz/util/APZThreadUtils.cpp
--- a/gfx/layers/apz/util/APZThreadUtils.cpp	Fri Jun 12 21:53:14 2020 +0000
+++ b/gfx/layers/apz/util/APZThreadUtils.cpp	Mon Jun 15 18:25:55 2020 +0000
@@ -4,11 +4,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "mozilla/layers/APZThreadUtils.h"
+#include "APZThreadUtils.h"
 
 #include "mozilla/ClearOnShutdown.h"
 #include "mozilla/StaticMutex.h"
 
+#include "nsISerialEventTarget.h"
+#include "nsThreadUtils.h"
+#include "nsXULAppAPI.h"
+
 namespace mozilla {
 namespace layers {
 